docker-compose.yml

```diff
@@ -7,5 +7,5 @@ services:
       - ./source:/source
     env_file: .env
     mem_limit: 7G
-    entrypoint: /bin/sh
-    command: -c "$$(wandb sweep sweep.yaml 2>&1  | tail -1 | cut -d ':' -f 3)"
+    entrypoint: /bin/bash
+    command: -c "source init_sweep.sh && wandb agent deepform/deepform/$$WANDB_SWEEP_ID"
```

Comment:
Why do you prefer running two commands here to putting the second command in the shell script? I would like to be able to run `./init_sweep.sh` to start a sweep (outside the normal docker workflow). Is that a bad idea for some reason?

Reply:
I think perhaps I had some future proofing in mind. If the sweep initialization is an independent action, then it becomes easier to initialize a sweep and then start up multiple agents to participate in the sweep. Until we do that though we could make our lives simpler and fold the agent starting into the script.

Reply:
Oh, is that how it's intended to work, that you send that token to a bunch of different machines and have them all start up agents to work on the same sweep? I can see that then.

I think I'd still like it in the same command for now? Even if we're running in a swarm, I still would prefer to have an orchestration tool activate all the agents rather than manually running a command on each of them, so that would still require a change.